src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -406,15 +410,16 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
         proposals: List[(BlockHash, Option[ProtocolId])],
         listings: List[(Block, List[Voting.BakerRolls])],
         ballots: List[(Block, List[Voting.Ballot])],
-        ballotCounts: List[(Block, Option[Voting.BallotCounts])]
+        ballotCounts: List[(Block, Voting.BallotCounts)]
     ): List[
       (BlockData, Option[ProtocolId], List[Voting.BakerRolls], List[Voting.Ballot], Option[Voting.BallotCounts])
     ] = blockData.map { bd =>
       val hash = bd.hash
+
       val proposal = proposals.find { case (blockHash, _) => blockHash == hash }.flatMap(_._2)
       val listing = listings.find { case (block, _) => block.data.hash == hash }.toList.flatMap(_._2)
       val ballot = ballots.find { case (block, _) => block.data.hash == hash }.toList.flatMap(_._2)
-      val ballotCount = ballotCounts.find { case (block, _) => block.data.hash == hash }.flatMap(_._2)
+      val ballotCount = ballotCounts.find { case (block, _) => block.data.hash == hash }.map(_._2)
```

Comment:
Looking at how they're used, I suspect that all those lists should be exposed as Map[BlockHash, Value]?

Reply:
changed to maps, not exactly [BlockHash, Value], but still it simplified a bit